File: Code/3DMANAGER-APP/3dmanager-app.client/src/models/print/PrintRequest.ts
export interface PrintRequest {
    groupId: number;
    userId: number;
    printName: string;
    printState: number;
    printPrinter: number;
    printFilament: number;
    printDescription: string;
    printTime: number;
    printRealTime: number;
    printFilamentUsed: number;
    imageFile?: File | null;
}

File: Code/3DMANAGER-APP/3DMANAGER-APP.BLL/Models/Print/PrintRequest.cs
using Microsoft.AspNetCore.Http;

namespace _3DMANAGER_APP.BLL.Models.Print
{
    public class PrintRequest
    {
        public int GroupId { get; set; }
        public int UserId { get; set; }
        public string PrintName { get; set; }
        public int PrintState { get; set; }
        public int PrintPrinter { get; set; }
        public int PrintFilament { get; set; }
        public string PrintDescription { get; set; }
        public int PrintTime { get; set; }
        public decimal PrintFilamentUsed { get; set; }
        public int PrintRealTime { get; set; }
        public IFormFile? ImageFile { get; set; }
    }
}
